### PR TITLE
Microsoft.VCRedist.2015+.x86 version 14.34.31931

### PR DESCRIPTION
**Pull request by TheBossMagnus**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/88244)

### CODE REVIEW

File: manifests/m/Microsoft/VCRedist/2015+/x86/14.34.31931/Microsoft.VCRedist.2015+.x86.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Microsoft.VCRedist.2015+.x86
+PackageVersion: 14.34.31931
```
Comment (Trenly) [inline manifests/m/Microsoft/VCRedist/2015+/x86/14.34.31931/Microsoft.VCRedist.2015+.x86.installer.yaml:5]:
```suggestion
PackageVersion: 14.34.31931.0
```

```
--> Comparing ARP Entries

DisplayName                                                        DisplayVersion Publisher             ProductCode
-----------                                                        -------------- ---------             -----------
Microsoft Visual C++ 2015-2022 Redistributable (x86) - 14.34.31931 14.34.31931.0  Microsoft Corporation {6ba9fb5e-8366-4cc4-bf65-25fe9819b2fc}
```

File: manifests/m/Microsoft/VCRedist/2015+/x86/14.34.31931/Microsoft.VCRedist.2015+.x86.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+InstallerSwitches:
+  Silent: /quiet /install
+  SilentWithProgress: /passive /install
+  Upgrade: /norestart
+  Custom: /norestart
+InstallerSuccessCodes:
+- 3010
+UpgradeBehavior: install
+AppsAndFeaturesEntries:
+- DisplayName: Microsoft Visual C++ 2015-2022 Redistributable (x86) - 14.34.31823
```
Comment (Trenly) [inline manifests/m/Microsoft/VCRedist/2015+/x86/14.34.31931/Microsoft.VCRedist.2015+.x86.installer.yaml:23]:
```suggestion
- DisplayName: Microsoft Visual C++ 2015-2022 Redistributable (x86) - 14.34.31931
```

File: manifests/m/Microsoft/VCRedist/2015+/x86/14.34.31931/Microsoft.VCRedist.2015+.x86.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  Publisher: Microsoft Corporation
+  ProductCode: '{485c6580-376a-450b-9a80-43c390b968a3}'
```
Comment (Trenly) [inline manifests/m/Microsoft/VCRedist/2015+/x86/14.34.31931/Microsoft.VCRedist.2015+.x86.installer.yaml:25]:
```suggestion
  ProductCode: '{6ba9fb5e-8366-4cc4-bf65-25fe9819b2fc}'

```

File: manifests/m/Microsoft/VCRedist/2015+/x86/14.34.31931/Microsoft.VCRedist.2015+.x86.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+Installers:
+- Architecture: x86
+  InstallerType: burn
+  InstallerUrl: https://download.visualstudio.microsoft.com/download/pr/6a4c74cd-8ee0-4757-9620-a11a5b48b1a7/CE4843A946EE3732EB2BFC098DB5741DC5495C7BEA204E11D379336DCC68E875/VC_redist.x86.exe
+  InstallerSha256: CE4843A946EE3732EB2BFC098DB5741DC5495C7BEA204E11D379336DCC68E875
+  ProductCode: '{485c6580-376a-450b-9a80-43c390b968a3}'
```
Comment (Trenly) [inline manifests/m/Microsoft/VCRedist/2015+/x86/14.34.31931/Microsoft.VCRedist.2015+.x86.installer.yaml:31]:
```suggestion
  ProductCode: '{6ba9fb5e-8366-4cc4-bf65-25fe9819b2fc}'
```

**Review by Trenly — CHANGES_REQUESTED** [review]

File: manifests/m/Microsoft/VCRedist/2015+/x86/14.34.31931/Microsoft.VCRedist.2015+.x86.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,26 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Microsoft.VCRedist.2015+.x86
+PackageVersion: 14.34.31931
```
Comment (mdanish-kh) [inline manifests/m/Microsoft/VCRedist/2015+/x86/14.34.31931/Microsoft.VCRedist.2015+.x86.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 14.34.31931.0
```
Must match Control Panel.

File: manifests/m/Microsoft/VCRedist/2015+/x86/14.34.31931/Microsoft.VCRedist.2015+.x86.yaml [hunk 6]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: Microsoft.VCRedist.2015+.x86
+PackageVersion: 14.34.31931
```
Comment (mdanish-kh) [inline manifests/m/Microsoft/VCRedist/2015+/x86/14.34.31931/Microsoft.VCRedist.2015+.x86.yaml:5]:
```suggestion
PackageVersion: 14.34.31931.0
```
Must match Control Panel.

File: manifests/m/Microsoft/VCRedist/2015+/x86/14.34.31931/Microsoft.VCRedist.2015+.x86.installer.yaml [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+  Upgrade: /norestart
+  Custom: /norestart
+InstallerSuccessCodes:
+- 3010
+UpgradeBehavior: install
+AppsAndFeaturesEntries:
+- DisplayName: Microsoft Visual C++ 2015-2022 Redistributable (x86) - 14.34.31931
+  Publisher: Microsoft Corporation
+  ProductCode: '{6ba9fb5e-8366-4cc4-bf65-25fe9819b2fc}'
+
```
Comment (mdanish-kh) [inline manifests/m/Microsoft/VCRedist/2015+/x86/14.34.31931/Microsoft.VCRedist.2015+.x86.installer.yaml:26]:
```suggestion
```
Might wanna remove this unnecessary line.

**Review by mdanish-kh — CHANGES_REQUESTED** [review]

**Review by mdanish-kh — CHANGES_REQUESTED** [review]
Please also change folder name inside directory `manifests/m/Microsoft/VCRedist/2015+/x86/` from `14.34.31931` to `14.34.31931.0`

File: manifests/m/Microsoft/VCRedist/2015+/x86/14.34.31931.0/Microsoft.VCRedist.2015+.x86.installer.yaml [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+InstallerSwitches:
+  Silent: /quiet /install
+  SilentWithProgress: /passive /install
+  Upgrade: /norestart
+  Custom: /norestart
+InstallerSuccessCodes:
+- 3010
+UpgradeBehavior: install
+AppsAndFeaturesEntries:
+- DisplayName: Microsoft Visual C++ 2015-2022 Redistributable (x86) - 14.34.31931.0
```
Comment (mdanish-kh) [inline manifests/m/Microsoft/VCRedist/2015+/x86/14.34.31931.0/Microsoft.VCRedist.2015+.x86.installer.yaml:23]:
```suggestion
- DisplayName: Microsoft Visual C++ 2015-2022 Redistributable (x86) - 14.34.31931
```
Must match Control Panel
image

**Review by mdanish-kh — CHANGES_REQUESTED** [review]

**Review by mdanish-kh — APPROVED** [review]

File: manifests/m/Microsoft/VCRedist/2015+/x86/14.34.31931.0/Microsoft.VCRedist.2015+.x86.installer.yaml [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Silent: /quiet /install
+  SilentWithProgress: /passive /install
+  Upgrade: /norestart
+  Custom: /norestart
+InstallerSuccessCodes:
+- 3010
+UpgradeBehavior: install
```
Comment (fernandog) [inline manifests/m/Microsoft/VCRedist/2015+/x86/14.34.31931.0/Microsoft.VCRedist.2015+.x86.installer.yaml:21]: @TheBossMagnus shouldn't this be set to `UpgradeBehavior: uninstallPrevious`

**Pull request merged**